src/PileupTrack.js

```diff
@@ -635,16 +651,36 @@ varying vec4 vColor;
           this.tilesetInfo,
         );
 
-        const DEFAULT_MAX_TILE_WIDTH = 2e5;
-
         if (
           tileWidth >
-          (this.tilesetInfo.max_tile_width || DEFAULT_MAX_TILE_WIDTH)
+          (this.tilesetInfo.max_tile_width || this.options.maxTileWidth)
```

Comment:
```suggestion
          (this.tilesetInfo.max_tile_width || this.options.maxTileWidth || DEFAULT_MAX_TILE_WIDTH)
```

In case for whatever reason the option is not set.